.github/workflows/ci.yml

```diff
@@ -162,6 +167,19 @@ jobs:
     secrets:
       earthly_runner_address: ${{ secrets.earthly_runner_address }}
       earthly_runner_secret: ${{ secrets.earthly_runner_secret }}
+  nightly:
+    uses: ./.github/workflows/pages.yml
```

Comment:
you should use  ` uses: ./.github/workflows/run.yml` here. Take a look on how the test job works. The workflow run does the discovery of all the targets in the repo earthfiles. Instead the docs target calls the specifid target in the specific earthfile